index.ts

```diff
@@ -12,7 +12,10 @@ async function addComment(client: Client, taskId: string, comment: string): Prom
 
 function getPreviousText(): {text?: string} {
   if (context.payload.action === 'edited') {
-    return {text: context.payload.changes.body.from};
+    const text = context.payload.changes?.body?.from;
+    if (text) {
+      return {text};
```

Comment:
Couldn't you always return this since `text` will be `string | undefined`?

Reply:
We could. Guess I'm slightly irked by objects like `{text: undefined}` 😄